server/lib/client-html.ts

```diff
@@ -549,6 +549,7 @@ class ClientHtml {
 
 function sendHTML (html: string, res: express.Response) {
   res.set('Content-Type', 'text/html; charset=UTF-8')
+  res.set('Vary', 'Accept-Language')
```

Comment:
Embed page does not vary so I don't think we should set the header here.

Reply:
Can we add `i18n: bool` as an argument to sendHTML?

Reply:
Sure, or maybe `localizedHTML: bool`